package.json

```diff
@@ -25,6 +25,7 @@
     "ember-cli": "2.8.0",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-babel": "^5.1.6",
+    "ember-cli-datepicker": "2.0.1",
```

Comment:
I don't think so. I think just admin can set what they are seeing on the screen. But I realized if we're saving the date as only "Mon" in the database we can't actually use this to search for dates b/c it needs an actual date.